ST_mortality.c

```diff
@@ -348,33 +348,31 @@ void grazing_EndOfYear( void){
 		IntU grazingyr =0;
 		g = RGroup[rg];
 
-		//printf("inside grazing_EndOfYear() year=%d, rgroupName=%s, grazingfreq_startyr=%d, grazingfreq=%d, proportionGrazing=%f startYear=%d \n",Globals.currYear,g->name,g->grazingfreq_startyr,g->grazingfrq,g->proportion_grazing, RGroup[rg]->startyr);
-
-		if (Globals.currYear < RGroup[rg]->startyr)
+		if (Globals->currYear < RGroup[rg]->startyr)
 		{
 			/* Grazing cannot occur for an RGroup[rg] until the year that RGroup[rg] is turned on */
 			continue;
 		}
 
-		if ((Globals.currYear >=g->grazingfreq_startyr) && (g->grazingfrq > 0))
+		if ((Globals->currYear >=g->grazingfreq_startyr) && (g->grazingfrq > 0))
 		{
 			if (g->grazingfrq < 1.0)
 			{
 				if (RandUni(&mortality_rng) <= g->grazingfrq)
 				{
-					grazingyr = Globals.currYear;
+					grazingyr = Globals->currYear;
 				}
 
 			}
-			else if (((Globals.currYear - g->grazingfreq_startyr) % (IntU) g->grazingfrq) == 0)
+			else if (((Globals->currYear - g->grazingfreq_startyr) % (IntU) g->grazingfrq) == 0)
 			{
-				grazingyr = Globals.currYear;
+				grazingyr = Globals->currYear;
 			}
 
 		}
 
 		//Implement grazing if this year is a year where grazing should occur
-		if (Globals.currYear == grazingyr)
+		if (Globals->currYear == grazingyr)
 		{
 			//printf( "currYear is equal to grazingYear so will iterate all the Species for doing grazing, RGroup[g]->est_count =%d \n",RGroup[rg]->est_count);
```

Comment:
This line can probably safely be removed.

Reply:
Resolved by 2a1ff37.